packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -296,7 +296,7 @@ class _BottomSheetState extends State<BottomSheet> {
       dragHandleMaterialState.remove(MaterialState.dragged);
     });
     bool isClosing = false;
-    if (details.velocity.pixelsPerSecond.dy > _minFlingVelocity) {
+    if (details.velocity.pixelsPerSecond.dy.abs() > _minFlingVelocity) {
```

Comment:
Only swiping down (positive y) velocity was considered even though the following code was expecting both directions as valid